keel-core/src/main/kotlin/com/netflix/spinnaker/keel/jackson/mixins/DeliveryArtifactMixin.kt

```diff
@@ -1,13 +1,23 @@
 package com.netflix.spinnaker.keel.jackson.mixins
 
+import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.annotation.JsonProperty.Access.WRITE_ONLY
-import com.netflix.spinnaker.keel.api.artifacts.VersioningStrategy
+import com.netflix.spinnaker.keel.api.artifacts.SortingStrategy
 
 internal interface DeliveryArtifactMixin {
   @get:JsonProperty(access = WRITE_ONLY)
   val deliveryConfigName: String
 
-  @get:JsonProperty(access = WRITE_ONLY)
-  val versioningStrategy: VersioningStrategy
+  @get:JsonIgnore
```

Comment:
We were already skipping this property when serializing. This just makes it explicit that we're not expecting users to specify the sorting strategy in the delivery config for the time being.

Reply:
I'm confused, I thought you had to specify write only in order for it to not serialize on read?

Reply:
`READ_ONLY` is "ignore when deserializing" (i.e. "reading" the object), `WRITE_ONLY` is "ignore when serializing" (writing the object to JSON), and `@JsonIgnore` is both. Not confusing at all, right? 🤦‍♂️